include/gridtools/stencil_composition/sid/concept.hpp

```diff
@@ -125,6 +125,8 @@
  *
  *  - Stride sid::get_stride<I>(Strides)
  *
+ * TODO(anstaf): add lower/upper bounds stuff when the API will be finally settled.
```

Comment:
Still true?

Reply:
sorry, I meant to add documentation on it.